samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/AuthorMessages_ar.properties

```diff
@@ -520,7 +520,7 @@ random_draw_msg_no_date=The questions for this part are generated randomly from
 random_update_questions=Update Questions
 
 # for upload questions in editAssessment.jsp
-upload_instruction=Click "Browse" to locate your file and then click "Upload" to upload your file.
+upload_instruction=Choose a file to upload.
```

Comment:
If these strings aren't translated in their respective languages, you can just remove them from the files. It will fall back to the default language anyway. Having these keys in the other language files may make it seem like they're already translated, and then they won't ever get translated.

Reply:
Okay. Thanks.